charts/managed-identity-wallets/values.yaml

```diff
@@ -218,7 +220,7 @@ datapool:
 managedIdentityWallets:
   secret:
     # -- Database connection string to the Postgres database of MIW
-    jdbcurl: "jdbc:postgresql://postgresql:5432/postgres?user=postgres&password=postgres"
+    jdbcurl: "jdbc:postgresql://managed-identity-wallets-postgresql:5432/postgres?user=postgres&password=postgres"
```

Comment:
Do we still need this if it is created dynamically in the helpers.tpl? If only needed as a placeholder, we should comment it and change the value to a clear placeholder value

Reply:
a placeholder if a custom url is required